cocos/platform/android/jni/JniHelper.cpp

```diff
@@ -49,9 +49,10 @@ jclass _getClassID(const char *className) {
                                                    cocos2d::JniHelper::loadclassMethod_methodID,
                                                    _jstrClassName);
 
-    if (nullptr == _clazz) {
+    if (nullptr == _clazz || env->ExceptionCheck()) {
```

Comment:
It seems that  the class is found and some exception occurs. Would you share what kind of exception did you met?